docs/user/kafka/confluent.rst

```diff
@@ -7,11 +7,189 @@ Confluent Integration
+``multipolygon``         A ``MultiPolygon``
+``geometrycollection``   A ``GeometryCollection``
+======================== ============================
+
+``geomesa.geom.default``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field represents the default ``Geometry`` for this ``SimpleFeatureType``. If the keys
+``geomesa.geom.format`` and ``geomesa.geom.type`` are not present on the same schema field, this attribute will
+be ignored.
+
+=========== ===============================
+Value       Description
+=========== ===============================
+``true``    The default ``Geometry``
+``false``   Not the default ``Geometry``
+=========== ===============================
+
+``geomesa.geom.srid``
+^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field represents a ``[[Geometry]]`` with the given spatial reference identifier (SRID).
+If the keys ``geomesa.geom.format`` and ``geomesa.geom.type`` are not present on the same schema field, this
+attribute will be ignored.
+
+=========== ==============================================================
+Value       Description
+=========== ==============================================================
+``4326``    Longitude/latitude coordinate system on the WGS84 ellipsoid
+=========== ==============================================================
+
+``geomesa.date.format``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be interpreted as a ``Date`` in the given format.
+
+=========================== ===================== ====================================================
+Value                       Schema Field Type     Description
+=========================== ===================== ====================================================
+``epoch-millis``            ``LONG``              Milliseconds since the Unix epoch
+``iso-date``                ``STRING``            Date format ``yyyy-MM-dd``
+``iso-instant``             ``STRING``            Date format ``yyyy-MM-dd'T'HH:mm:ss.SSSZZ``
+``iso-instant-no-millis``   ``STRING``            Date format ``yyyy-MM-dd'T'HH:mm:ssZZ``
+=========================== ===================== ====================================================
+
+``geomesa.date.default``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field represents the default ``Date`` for this ``SimpleFeatureType``. If the key
+``geomesa.date.format`` is not present on the same schema field, this attribute will be ignored.
+
+=========== =============================
+Value       Description
+=========== =============================
+``true``    The default ``Date``
+``false``   Not the default ``Date``
+=========== =============================
+
+``geomesa.index``
+^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be indexed by GeoMesa.
+
+=========== ==================================
+Value       Description
+=========== ==================================
+``true``    Index this attribute
+``false``   Do not index this attribute
+``full``    Index the full ``SimpleFeature``
```

Comment:
What indexing are we talking about?  KDS or big-data-backend?  What does the full do?!

Reply:
No idea what it does but it is briefly mentioned here: https://www.geomesa.org/documentation/stable/user/accumulo/index_config.html. And it was in one of the SFTs that this was going to be used for so I figured I'd include it?

Reply:
do the avro schemas support setting key-values at the schema level, as well as per-attribute? it might make sense to not explicitly document/check every single key value pair, but just set them as is in the simple feature type or attribute descriptor. then, we won't have to modify any code when we add new ones, or if someone wants to use a schema with accumulo vs kafka, etc. (this index flag is not relevant for kafka, only accumulo, so it's a bit confusing to add and document it).

Reply:
Gotcha, so just put any extra props on the schema in the SFT/attribute user data, and only document those that are relevant to deserialization, e.g. geometry type, geometry format, date format?

Reply:
Fixed